src/app/modules/core/components/jobsons/jobsons.component.ts

```diff
@@ -167,33 +170,59 @@ export class JobsonsModalComponent implements OnInit {
     } else {
       this.dateModel = new Date(this.dateModel);
     }
-    this.TravelTimeService.ExecuteJobson(this.StudyService.selectedStudy.SpillMass, this.dateModel.toISOString(), this.reachList)
+    let postReachList = [];
+    this.reachList.forEach(reach => {
+      reach.parameters.splice(6,1);
+      postReachList.push(reach);
+    })
+    this.TravelTimeService.ExecuteJobson(this.StudyService.selectedStudy.SpillMass, this.dateModel.toISOString(), postReachList)
       .toPromise().then(data => {
-        console.log(data);
         this.StudyService.selectedStudy.Results = data;
         this.StudyService.SetWorkFlow("totResults", true);
         this.gettingResults = false;
+        this.activeModal.dismiss();
       })
       .catch((err) => {
         console.log("error: ", err.message);
         this.sm(err, "Time of Travel Services")
         this.gettingResults = false;
       });
   }
+
+  public openCloseItem(index): void {
+    this.currentIndex = index;    
+    if(this.reachList[this.currentIndex].parameters.show) {
```

Comment:
Here you forgot to open the last one if the new one closed, and if last one is different from current. But similarly to current one, your last one can be a null, so you have to check it too.